doc/installation.rst

```diff
@@ -8,36 +8,95 @@
+Download and install ``git`` to your system using the following `link <https://git-scm.com/downloads>`__.
+It is highly recommended to `learn the git basics <https://git-scm.com/doc>`__.
+
+VSCode
+------
+In order to write and debug python code, you need an Integrated Development Environment (IDE) that is a software used to write code. We recommend `Visual Studio Code <https://code.visualstudio.com/>`_, which is freely available online and provides an easy to use interface with Git. Obviously, any alternatives like `PyCharm <https://www.jetbrains.com/pycharm/>`_ or `Sublime <https://www.sublimetext.com/>`_ will work as well.
+
+Java
+----
+PyPSA-Earth currently needs Java redistribution to work properly. To check if Java is still installed you can request it's version from a terminal:
+
+.. code:: bash
+
+    java --version
+
+The expected output should resemble the following:
+
+.. code:: bash
+
+    java version "1.8.0_341"
+    Java(TM) SE Runtime Environment (build 1.8.0_341-b10)
+    Java HotSpot(TM) 64-Bit Server VM (build 25.341-b10, mixed mode)
+
+In case you don't have Java, you have to install it from the `official website <https://www.oracle.com/java/technologies/downloads/>`_ or equivalent. Depending on the version of OS, download and install ``JDK 21`` or ``JDK 17`` from the link.
+
+
+PyPSA-Earth Installation
+========================
 
 Clone the Repository
-====================
+--------------------
 .. note::
 
-  In order to work with the provided Jupyter notebooks in the `documentation repository <https://github.com/pypsa-meets-earth/documentation>`_, it is recommended to follow the folder structure suggested in :ref:`notebooks`.
+  In order to work with the provided Jupyter notebooks in the `documentation repository <https://github.com/pypsa-meets-earth/documentation>`__, it is recommended to follow the folder structure suggested in :ref:`notebooks`.
 
-First of all, clone the `PyPSA-Earth repository <https://github.com/pypsa-meets-earth/pypsa-earth/>`_ using the version control system ``git``.
+First of all, clone the `PyPSA-Earth repository <https://github.com/pypsa-meets-earth/pypsa-earth/>`__ using the version control system ``git``.
 The path to the directory into which the ``git repository`` is cloned, must **not** have any spaces.
-If you do not have ``git`` installed, follow installation instructions `here <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_.
+The following commands can be executed in command prompt of ``miniconda``, terminal of ``VSCode``, or in ``Git Bash``.
 
 .. code:: bash
 
     /some/other/path % cd /some/path/without/spaces
 
     /some/path/without/spaces % git clone https://github.com/pypsa-meets-earth/pypsa-earth.git
 
-.. _deps:
+For more details regarding the folder structure, visit :ref:`notebooks` section.
```

Comment:
Absolutely agree that it would be very nice to give some hints on the project architecture, but not sure `notebooks` section would give very detailed insights for folders structure. Although, I fear that at the time being there is no an up-to-date source describing the folders structure. There is a nice description in 
https://github.com/pypsa-meets-earth/documentation/blob/main/hackathon/1_architecture_pypsaafrica.ipynb, but it's outdated

Reply:
Hi, @ekatef. `notebooks.rst` is basically "Examples" section in the documentation. It just gives hint to how to structure `documentation` and `pypsa-earth` to directly use notebooks in documentation. But when I have installed PyPSA-Earth initially I did not followed that structure and I did not even read it fully (that structure part). I just changed path in documentation notebooks to use them. So I guess for the new user it is a high priority to run the model first and then evaluate it. And I do not think that majority will think about documentation before running the model (as I did). Of course, I have referred to that page now, the content of which we can change as we think is better. 

Or alternatively, we can remove that folder structure and just to refer to `documentation` that it has useful notebooks. Users themselves change the path and etc (because even the structure is conserved, those codes needs some minor fix yet).

Reply:
Ahh, right! Ok, thanks for the explanation. 

I do agree that's definitely not the first thought to look into the notebooks, when installing the model. I'd opt for the second option you suggest, namely to remove mentioning of the folders structure and refer to `documentation`. Would you agree?

Reply:
Super, very clear now.